tests/bson/bug0592.phpt

```diff
@@ -32,6 +32,7 @@ foreach ($tests as $json) {
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
+%a
```

Comment:
What is being printed before the first test case? It may make more sense to simply enable and disable `mongodb.debug=stderr` around the `foreach` that executes all of the test cases, instead of having the INI setting on for the entire process. We really only want to check for the overflow warnings.